doc/tutorials/dnn/table_of_content_dnn.markdown

```diff
@@ -0,0 +1,35 @@
+The dnn Functionality (dnn module) {#tutorial_table_of_content_dnn}
+=====================================
+
+Here you will learn the about the basic building blocks of the library. A must read and know for
```

Comment:
Probably you want to update also the description for these tutorials (here and below)?